src/main/java/frc/team2412/robot/subsystems/VisionSubsystem.java

```diff
@@ -73,17 +75,16 @@ public VisionSubsystem(BiConsumer<Pose2d, Double> poseConsumer) {
 	}
 
 	public void updateEvent(NetworkTableEvent event) {
-		latestResult = photonCamera.getLatestResult();
-		updatePoseCache();
-		if (robotPose != null) {
-			lastTimestampSeconds = latestResult.getTimestampSeconds();
-			poseConsumer.accept(robotPose.toPose2d(), lastTimestampSeconds);
+		latestPose = photonPoseEstimator.update();
```

Comment:
Any chance the latest available can be too outdated for us want to use?

Reply:
iirc we're just keeping the latest pose in that variable in case someone else wants to use it in the future (nothing is being done with it right now)

When the pose gets sent to drivebase it also has a timestamp attached to it so I'd *assume* the pose estimator weighs the pose less and less as it gets more and more out of date